scripts/constraints.txt

```diff
@@ -0,0 +1,205 @@
+#
+# This file is autogenerated by pip-compile
+# To update, run:
+#
+#    pip-compile requirements.txt
+#
```

Comment:
Comment should be in `requirements.txt` file

Reply:
This is the generated file. Fixed the command line to the right output file.